src/steam_network/protocol/protobuf_socket_handler.py

```diff
@@ -249,7 +207,7 @@ async def TokenLogOn(self, account_name: str, steam_id: int, access_token: str,
         message.qos_level = 3
         message.machine_id = machine_id
         message.account_name = account_name
-        #message.password = ""
+        # message.password = ""
```

Comment:
if possible, maybe make exceptions to the spacing rule for commented out code. if you uncomment this, it'll cause an error due to unmatched spacing.

Reply:
That's something that your code editor does wrong then ;) 
Personal take: I'd strongly recommend to use vscode over VS for editing Python code. Especially since there you can add things like flake8 (and autopep8/black if you like) which can visually assist with the pep thing already at editing time and also outlines when there are unused variables, functions to non-existing function etc